include/boost/geometry/algorithms/detail/overlay/get_distance_measure.hpp

```diff
@@ -61,13 +61,23 @@ namespace detail_dispatch
 
 template <typename CalculationType, typename CsTag>
 struct get_distance_measure
-        : not_implemented<CsTag>
```

Comment:
This struct is now going to be instantiated for all geometries defining CSTag other than cartesian_tag, also for cs_undefined_tag regardless of the strategy passed by the user. This is incorrect.